addons/web/static/src/js/views/basic/basic_model.js

```diff
@@ -1322,10 +1322,15 @@ var BasicModel = AbstractModel.extend({
                         delete record._warning;
                         onchangeDef.resolve(_.keys(changes).concat(Object.keys(result && result.value || {})));
                     }).fail(function () {
-                        self._visitChildren(record, function (elem) {
-                            _.extend(elem, initialData[elem.id]);
-                        });
-                        onchangeDef.resolve({});
+                        // safe fix for stable version, for opw-2267444
+                        if (!options.force_fail) {
+                            self._visitChildren(record, function (elem) {
```

Comment:
you must reset the initial data in any cases whatever the value of the force_fail option, I think

Reply:
besides that, I agree with the fix. Don't forget to remove the option in the master forwardport

Reply:
it was probably because in the kanban case we reloaded on fail, but since it could be used (before master) elsewhere I did the change